nautobot/ipam/models.py

```diff
@@ -135,6 +135,12 @@ class VRF(PrimaryModel):
         through="ipam.VRFDeviceAssignment",
         through_fields=("vrf", "virtual_machine"),
     )
+    virtual_device_contexts = models.ManyToManyField(
```

Comment:
Can we augment the relevant factory(s) to populate this data?

Reply:
Going to add a followup issue to augment VRF factory to include `devices` and `virtual_machines` @post_generation as well

Reply:
https://github.com/nautobot/nautobot/issues/6938